build.rs

```diff
@@ -289,7 +289,7 @@ pub fn fold_binop(p: &mut MutatorPlugin, id: NodeId, op: BinOp, left: P<Expr>, r
                 );
             quote_expr!(p.cx(), {{
                 ::mutagen::report_coverage($n..$current, &$sym[$flag], $mask);
-                ::mutagen::gt($right, $left, $n)
+                ::mutagen::gt(&$right, &$left, $n)
```

Comment:
Which lint does this appease?

Reply:
Ah, `needless_pass_by_value`. OK then.

Reply:
Sorry, I missed the notification about the comment above.